src/site/content/en/blog/introducing-tooling-report/index.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
We usually want the URL to match the title but this title is too long for that, so it seems like `https://web.dev/introducing-tooling-report` is good? We might not want to save `https://web.dev/tooling-report` for something else in the future

Reply:
This will go live under `/blog/introducing-tooling-report`, right? That’d work for me.

Reply:
We have flat URLs so it will be `https://web.dev/introducing-tooling-report` (no `/blog/`)